### ISSUE TITLE
LANTERN NEED CUSTOMERS CONTACT SUPPORT

### ISSUE THREAD

**Issue by tedcandy** [getlantern/lantern#7602]
LANTERN NEED CUSTOMERS CONTACT SUPPORT, and more import to reply to customers report an issue...

**Comment by shelldu** [comment]
I don't think you have read the agreement when you purchase Lantern pro.The customer service is very limited and you cannot refund for any reason, Github is the main channel that you can ask for any technical support.If you have any issue, just post it on Github, the administrator will see and reply it to you, despite sometimes it may take several days; or someone else may help you to solve your problem.

用我的邀请码YJ2MPSM，可赠送3个月时长。
